snippets/collection/sidebar.liquid

```diff
@@ -2,7 +2,9 @@
     {% include 'collection/sidebar/applied-filters' %}
     {% include 'collection/sidebar/keyword' %}
     {% if customer and customer.is_registered_user %}
-        {% include 'collection/sidebar/previously-purchased' %}
+        {% unless customer.is_first_time_buyer %}
```

Comment:
This statement could be simplified and unioned with if above like that: customer.is_first_time_buyer != true

Reply:
i think it's just a preference. 'unless' works the same as '!="
https://github.com/Shopify/liquid/issues/138

Reply:
Yes, but instead of two lines with the statement, we could have just one.

Reply:
done